config/env.staging

```diff
@@ -167,6 +167,17 @@ PARSER_URL=https://dev.api.telescope.cdot.systems/v1/parser
 PLANET_PORT=9876
 
 
+################################################################################
+# Docusaurus App
+################################################################################
+
+# Docusaurus Server Port (default is 4631)
+DOCS_PORT=4631
```

Comment:
Same question here: can we remove this in staging?

Reply:
Removed :)